src/volatile_memory.rs

```diff
@@ -632,7 +726,7 @@ impl<'a, T: ByteValued> VolatileRef<'a, T> {
     }
```

Comment:
Should we check that the pointer satisfy the alignment requirement of T? read_volatile/write_volatile() requires the that the pointer is correctly aligned.

Reply:
No, the pointer is to `Packed<T>`, not `T`, and `Packed<T>` does not have alignment requirements because it's `#[repr(C, packed)]`.